gamemodes/terrortown/entities/entities/ttt_decoy.lua

```diff
@@ -18,13 +18,7 @@ ENT.CanUseKey = true
 function ENT:Initialize()
 	self:SetModel(self.Model)
 
-	if SERVER then
-		self:PhysicsInit(SOLID_VPHYSICS)
-	end
-
-	self:SetMoveType(MOVETYPE_VPHYSICS)
-	self:SetSolid(SOLID_VPHYSICS)
-	self:SetCollisionGroup(COLLISION_GROUP_INTERACTIVE)
```

Comment:
Then destruction won't work. I specificly chose the collision type so that it works as intended. Also the docy still works so I'm not seeing the problem here

Reply:
Allrighty just was making sure, and only noted this because i wanted to double check because this differed from the previous type

Reply:
that's fine! Sometimes I miss stuff and the right questions help us find issues. In this case I'm pretty sure my implementation is fine. I did change the physics and collision, but in-game it works the same so I guess this is fine